docs/hello_nextflow/02_hello_channels.md

```diff
@@ -0,0 +1,868 @@
+This is progress.
+
+### 1.4. Add the channel as input to the process call
+
+Now we need to actually plug our newly created channel into the `sayHello()` process call.
+
+In the workflow block, make the following code change:
+
+_Before:_
+
+```groovy title="hello-channels.nf" linenums="26"
+// emit a greeting
+sayHello()
+```
+
+_After:_
+
+```groovy title="hello-channels.nf" linenums="26"
+// emit a greeting
+sayHello(greeting_ch)
+```
+
+This tells Nextflow to run the `sayHello` process on the contents of the `greeting_ch` channel.
+
+### 1.5. Run the workflow command again
+
+Let's run it!
+
+```bash
+nextflow run hello-channels.nf
+```
+
+If you made all four edits correctly, you should get another successful execution:
+
+```console title="Output"
+ N E X T F L O W   ~  version 24.10.0
+
+ ┃ Launching `hello-channels.nf` [prickly_avogadro] DSL2 - revision: b58b6ab94b
+
+executor >  local (1)
+[1f/50efd5] sayHello (1) [100%] 1 of 1 ✔
+```
+
+Feel free to check the results directory to satisfy yourself that the outcome is still the same as previously.
+So far we're just progressively tweaking the code to increase the flexibility of our workflow while achieving the same end result.
+
+### Takeaway
+
+You know how to use a simple channel to provide an input to a process.
+
+### What's next?
+
+Learn how to make the workflow run on a batch of multiple input values.
+
+---
+
+## 2. Modify the workflow to run on a batch of input values
+
+Workflows typically run on batches of inputs that are meant to be processed in bulk, so we want to upgrade the workflow to accept multiple input values.
+
+### 2.1. Load multiple greetings into the input channel
+
+Conveniently, the `Channel.of()` channel factory we've been using is quite happy to accept more than one value, so we don't need to modify that at all.
+We just have to load more values into the channel.
+
+In the workflow block, make the following code change:
+
+_Before:_
+
+```groovy title="hello-channels.nf" linenums="46"
+// create a channel for inputs
+greeting_ch = Channel.of('Hello')
+```
+
+_After:_
+
+```groovy title="hello-channels.nf" linenums="46"
+// create a channel for inputs
+greeting_ch = Channel.of('Hello','Bonjour','Holà')
+```
+
+The documentation tells us this should work. Can it really be so simple?
+
+### 2.2. Run the command and look at the log output
+
+Let's try it.
+
+```bash
+nextflow run hello-channels.nf
+```
+
+Well, it certainly seems to run just fine:
+
+```console title="Output"
+ N E X T F L O W   ~  version 24.10.0
+
+ ┃ Launching `hello-channels.nf` [lonely_pare] DSL2 - revision: b9f1d96905
+
+executor >  local (3)
+[3d/1fe62c] sayHello (2)       [100%] 3 of 3 ✔
+```
+
+However... This seems to indicate that '3 of 3' calls were made for the process, which is encouraging, but this only give us one subdirectory path. What's going on?
```

Comment:
```suggestion
However... This seems to indicate that '3 of 3' calls were made for the process, which is encouraging, but this only shows us a single run of the process, with one subdirectory path (`3d/1fe62c...`). What's going on?
```

The previous one made me think I should be looking at the actual directories momentarily.